charts/stable/traefik/templates/middlewares/theme-park.yaml

```diff
@@ -0,0 +1,30 @@
+{{- $values := .Values }}
+{{- $namespace := ( printf "ix-%s" .Release.Name ) }}
+{{- if or ( not .Values.ingressClass.enabled ) ( and ( .Values.ingressClass.enabled ) ( .Values.ingressClass.isDefaultClass ) ) }}
+{{- $namespace = "default" }}
+{{- end }}
+{{ range $index, $middlewareData := .Values.middlewares.redirectScheme }}
+
+---
+# Declaring the user list
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: {{ $middlewareData.name }}
+  namespace: {{ $namespace }}
+spec:
+    plugin:
+        traefik-themepark:
+            app: {{ $middlewareData.app }}
+            theme: {{ $middlewareData.theme }}
+{{ end }}
+
+
+
+    name: my-traefik-themepark
+    namespace: my-namespace
+spec:
+    plugin:
+        traefik-themepark:
+            app: sonarr
+            theme: dark
```

Comment:
```suggestion
```

Reply:
LoL , copied it, to tranfser the structure and forgot, Thanks!